src/components/PhotoAlbum/PhotoAlbum.tsx

```diff
@@ -33,10 +33,11 @@ export const PhotoAlbum = ({ photos }: { photos: any[] }) => {
           distance={Math.abs(currentPhoto - index)}
           onClick={() => setCurrentPhoto(index)}
           active={(index === currentPhoto).toString()}
-          src={`https:${photo.fields.file.url}`}
-          height={300}
-          width={600}
+          src={`https:${photo.fields.file.url}?&w=500`}
+          height={280}
+          width={500}
           alt=""
+          quality={currentPhoto === index ? 100 : 10}
```

Comment:
Great addition!

What if we also used the [priority](https://nextjs.org/docs/pages/api-reference/components/image#priority) property?

Reply:
You mean, use it the same way we use the quality property? Very interesting, i can test that

Reply:
Exactly!